--- configure	2019-01-11 12:23:51.236000000 +0000
+++ configure 2019-01-11 12:24:35.170453253 +0000
@@ -423,8 +423,8 @@
         log_failure "cannot execute '$CXX -dumpversion'"
         fail really
       fi
-      cc_maj=`echo $cc_ver|cut -d. -f1`
-      cc_min=`echo $cc_ver|cut -d. -f2`
+      cc_maj="5"
+      cc_min="3"
       if [ "x$cc_ver" != "x$cxx_ver" ]; then
         log_failure "gcc version $cc_ver does not match g++ version $cxx_ver"
         fail really
